specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -934,7 +952,8 @@ model AcsChatThreadEventInThreadBaseProperties
 #suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
 model AcsChatMessageEventBaseProperties extends AcsChatEventBaseProperties {
   /** The chat message id */
-  messageId?: string;
+  @madeRequired(ServiceApiVersions.v2024_01_01)
```

Comment:
This doesn't need to be made required as it is a string. It should also be added to required list in the swagger.